src/client/automation/playback/type/type-text.js

```diff
@@ -219,12 +219,20 @@ function _typeTextToContentEditable (element, text) {
 }
 
 function _typeTextToTextEditable (element, text) {
-    const elementValue      = domUtils.getElementValue(element);
-    const textLength        = text.length;
-    let startSelection    = textSelection.getSelectionStart(element);
-    let endSelection      = textSelection.getSelectionEnd(element);
-    const isInputTypeNumber = domUtils.isInputElement(element) && element.type === 'number';
-    const needProcessInput  = simulateTextInput(element, text);
+    const elementValue         = domUtils.getElementValue(element);
+    const textLength           = text.length;
+    let startSelection         = textSelection.getSelectionStart(element);
+    let endSelection           = textSelection.getSelectionEnd(element);
+    const isInputTypeNumber    = domUtils.isInputElement(element) && element.type === 'number';
+    const needProcessTextInput = !browserUtils.isChrome || eventSimulator.beforeInput(element, text);
```

Comment:
I rarely review the automation codebase, but seriously, I have no faintest idea why we should check if the browser is chrome. If you find any kind of browser-specific behavior, please add a comment. It can save a lot of time later.
E.g., I'd like to see this link in the comment https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/beforeinput_event

Also, I think the pr summary should be obligatory.

Reply:
I added description to the first post.

>Also, I think the pr summary should be obligatory.

Of course,  I've missed it

Reply:
Thanks 👍